cosmos/evmos_9001.json

```diff
@@ -35,6 +35,12 @@
       "coinDecimals": 18,
       "coinGeckoId": "evmos",
       "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/evmos_9001/aevmos.png"
+    },
+    {
+      "coinDenom": "CROWDP",
+      "coinMinimalDenom": "erc20/0xfbF4318d24a93753F11d365A6dcF8b830e98Ab0F",
```

Comment:
Hey, sorry, i didn't look into your changes deeply.

Please verify whether the `coinMinimalDenom` of the currency is a denom used in the Cosmos node or the EVM node.
If it's Cosmos, it seems fine to merge.
If it's EVM, it should be updated to `erc20:0x---` format. Please make sure that it works on Keplr extension by doing tests on your local environment. You can find how to build the wallet in https://github.com/chainapsis/keplr-wallet?tab=readme-ov-file#building-browser-extension-locally